operators/vcp-operator/ci.yaml

```diff
@@ -1 +1,2 @@
 cert_project_id: 66228542f2903c1122304c1e
+updateGraph: semver-mode
```

Comment:
FYI this isn't valid here, you have to use valid `skips` or `replaces` in your CSV.